model_analyzer/analyzer.py

```diff
@@ -431,6 +431,7 @@ def _populate_search_parameters(self, client, gpus):
                 config=self._config,
                 model=model_profile_spec,
                 is_bls_model=bool(self._config.bls_composing_models),
+                is_ensemble_model=model_profile_spec.is_ensemble(),
```

Comment:
I'm surprised we don't have an enum for type/mode instead of multiple bools passed in.. unless we support BLS + ensemble at the same time?

Reply:
Yes, I could have probably done this with an enum. A model can either be BLS, ensemble, or vanilla.